### ISSUE TITLE
Sidekick alternative

### ISSUE THREAD

**Issue by VeeK727** [NativeScript/NativeScript#8983]
I miss the GUI terribly. Ever since Nativescript changed hands and Sidekick fell through the cracks I've been vocal about getting it back. I've talked to the devs about this before and they were kinda interested in making their own version of it but not all the way in.

So I'm creating this feature request here so others can chime in too and maybe +1 this post. If enough people show interest maybe they'll do it.

I want to list a few features in Sidekick that made my life easier because I didn't have to fumble with terminal commands. A sidekick replacement should have these: 

1. Project Management: Sidekick created tiles for each project and switching them was simple. Creating projects was easy too. You don't need to create the starting templates but blank will do. Sidekick would setup all the folder structure and files.

2. Plugins Management: Adding, Updating and Removing plugins from Sidekick is super easy. You can search from right within Sidekick and don't need the commands to install plugins.

3. Asset management: This is a biggy. Sidekick would take one app icon and splash screen and generate the images in all sizes and placed them in relevant folders. Thats quite a life saver.

4. Info and Permissions: Managing the app id, version, name and description with required permissions are all in the sidekick so you don't have to go about looking for relevant files to change this info. I think this section should get even more options like on every project I do [this](https://github.com/NativeScript/NativeScript/issues/5088#issuecomment-353394591). But I'm getting greedy here, whats already there is fine :)

5. Console output with the option to copy to clipboard is super helpful.

I think Sidekick was severely underrated by the Nativescript community, perhaps because the devs prefer CLI. But for regular people like me, who pretty much came to Nativescript because they wanted to make apps without learning 2 new languages - Sidekick is pretty close to essential. If anyone switches to Sidekick they wouldn't want to go back.

I hope you guys make your own version of it.

**Comment by crowmagnumb** [comment]
Re: Item 3

I use the commands ...
```
ns resources generate splashes <source.splash>
ns resources generate icons <source_icon>
```
... which I believe does exactly what you are asking.

**Comment by VeeK727** [comment]
Yes I know. There are of course CLI codes for doing all of this but my argument is for a GUI that makes all this easier. The same thing in Sidekick was delightful if you've used it.

**Comment by crowmagnumb** [comment]
Ah, sorry. I for one couldn't imagine desiring a GUI over the command-line.  :) You could always write a simple app with a button to shell out the command-line, no?  :)

**Comment by VeeK727** [comment]
No that would be outside my skill set and experience level. 

> I for one couldn't imagine desiring a GUI over the command-line
That is so 2003 lol. To each their own I guess. I loved Sidekick.

**Comment by DimitrisRK** [comment]
> No that would be outside my skill set and experience level.
> 
> > I for one couldn't imagine desiring a GUI over the command-line
> 
> That is so 2003 lol. To each their own I guess. I loved Sidekick.

Actually, CLI is not considered old school for some time now. It's faster than a GUI in terms of development since typing a command does what you want which can also be auto-typed depending on your terminal tools (e.g. fish shell).

Of course, I won't deny that a GUI would be a good addition for people who are used to working with it. Not really an individual application like Sidekick because it's an outdated concept, but a GUI extension/plugin installed on any popular IDE/Editor.

For example, in the case of Sublime Text 3 it could be a menu similar to GitSavvy:
![Screenshot from 2020-10-27 12-58-53](https://user-images.githubusercontent.com/55595100/97292722-480c0980-1854-11eb-8a0c-8c386eebcf6a.png)
